src/ui-manager.js

```diff
@@ -48,6 +50,9 @@ class UIManager {
   constructor(player: Player, config: Object) {
     this.player = player;
     this.config = config;
+    if (config.logLevel && this.LogLevel[config.logLevel]) {
+      setLogLevel(this.LogLevel[config.logLevel]);
```

Comment:
@OrenMe @dan-ziv - this line seems to be faulty. It is not possible to set config level now, as this line always evaluates to undefined (LogLevel is an ENUM that contains objects of {value: 1, name: "DEBUG"}, etc.. It does not contain values).

Reply:
<img width="568" alt="debug-issue" src="https://user-images.githubusercontent.com/150514/38171327-8ef235e8-35a0-11e8-9002-027800732a9f.png">